source/agora/common/BanManager.d

```diff
@@ -272,9 +277,9 @@ public class BanManager
 
     ***************************************************************************/
 
-    protected time_t getCurTime () @safe nothrow @nogc
+    protected time_t getCurTime () @safe nothrow
```

Comment:
Can't we add `@nogc` to the Clock instead?

Reply:
> Can't we add @nogc to the Clock instead?

That makes an error as follows.

source\agora\network\Clock.d(111,25): Error: `@nogc` function `agora.network.Clock.Clock.networkTime` cannot call non-@nogc function `std.conv.to!int.to!long.to`

Reply:
But that's `networkTime()`, not `localTime()`.

Reply:
Ah, Sorry. Another error also occurred.

source/agora/common/BanManager.d(245,56): Error: @nogc function agora.common.BanManager.BanManager.isBanned cannot call non-@nogc function agora.common.BanManager.BanManager.getCurTime

Reply:
But I'm saying keep `getCurTime` and `isBanned` as `@nogc` and mark `Clock.localTime` as `@nogc`. Shouldn't it work then?

Reply:
I see what you mean. I thought that the two functions, `localTime()` and `networkTime()` should have the same modifier. I will modify and check it. Thanks.